privaterelay/urls.py

```diff
@@ -44,19 +44,14 @@
     ),
     path('accounts/', include('allauth.urls')),
     path('api/', include('api.urls')),
+    # This route is still referred to in wrapped_email.html, so leave it in
+    # (even though the homepage is no longer rendered by Django):
     path('faq', views.faq, name='faq'),
+    # This route is still referred to in middleware.py, so leave it in
+    # (even though the homepage is no longer rendered by Django):
+    path('', views.home, name='home'),
 ]
 
-if not settings.SERVE_REACT:
-    urlpatterns += [
-        path('accounts/profile/', views.profile, name='profile'),
-        path('accounts/settings/', views.settings_view, name='settings'),
-        # This redirects users back to the homepage after updating settings
-        path('accounts/profile/settings_update', views.settings_update_view, name='settings-update'),
-        path('premium', views.premium_promo, name='premium-promo'),
-        path('', views.home, name='home'),
-    ]
-
```

Comment:
suggestion (non-blocking): Looks like we can also remove these entire view functions from `privaterelay.views`: `profile`, `settings_view`, `settings_update_view`, `premium_promo`. Since they are not bound to any

Reply:
Ah yes, good one: 42131f01

Reply:
We still need any patterns that are called by name by `reverse`. The ones I see left in the code are `faq`, `home`, `profile` (privaterelay/views.py line 58), `fxa_login`, and `relayaddress-detail`. So we need `profile` back.

Reply:
Sharp eye @jwhitlock! The `reverse` for the profile never gets called though, because the view it's in in turn only exists for the URL to the homepage, and never gets rendered. Possibly I could replace the remaining views by empty functions, i.e. removing this `reverse` call?

That said, it feels a bit weird to me to keep these views just to be able to call `reverse` on them to generate a URL - I think the idea behind `reverse` is to ensure links remain up-to-date even if URLs change? Which doesn't get achieved because the views don't represent "real" URLs. (Besides, [URLs should remain valid](https://www.w3.org/Provider/Style/URI.html) anyway.) How would you feel about replacing all `reverse` calls with direct references to the URLs they end up at?

Reply:
> How would you feel about replacing all reverse calls with direct references to the URLs they end up at?

I think that's a good plan, ship it!

Reply:
https://github.com/mozilla/fx-private-relay/pull/1999